### PR TITLE
[zh-cn]: update the translation of CSS `matrix3d()` transform function

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/matrix3d
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/transform-function/matrix3d/index.md
* Last commit: https://github.com/mdn/content/commit/802978f38824a4132b4f9b3d3c23fb6970beba74

### CODE REVIEW

File: files/zh-cn/web/css/transform-function/matrix3d/index.md [hunk 1]
```diff
@@ -1,92 +1,167 @@
...
 title: matrix3d()
 slug: Web/CSS/transform-function/matrix3d
+l10n:
+  sourceCommit: 802978f38824a4132b4f9b3d3c23fb6970beba74
 ---
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/Web/CSS) 函数 **`matrix3d()`** 以 4x4 齐次矩阵的形式定义一个 3D 转换。其结果是一个 {{cssxref("&lt;transform-function&gt;")}} 数据类型。
+**`matrix3d()`** [CSS](/zh-CN/docs/Web/CSS) [函数](/zh-CN/docs/Web/CSS/CSS_Functions)将 3D 变换定义为一个 4x4 齐次矩阵。其结果是一个 {{cssxref("&lt;transform-function&gt;")}} 数据类型。
```
Comment (yin1999) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:10]: `matrix3d()` 是用来定义 3D 变换的，不是用来定义矩阵的。翻译的时候还请参考一下原来的翻译。

```suggestion
**`matrix3d()`** [CSS](/zh-CN/docs/Web/CSS) [函数](/zh-CN/docs/Web/CSS/CSS_Functions)以 4x4 齐次矩阵的形式定义一个 3D 变换。其结果是一个 {{cssxref("&lt;transform-function&gt;")}} 数据类型。
```

File: files/zh-cn/web/css/transform-function/matrix3d/index.md [hunk 2]
```diff
@@ -1,92 +1,167 @@
...
-  - : Are {{cssxref("&lt;number&gt;")}}s describing the linear transformation.
+- _a1_ _b1_ _c1_ _d1_ _a2_ _b2_ _c2_ _d2_
+  _a3_ _b3_ _c3_ _d3_
+  - : 表示线性变换的 {{cssxref("&lt;number&gt;")}} 值。
 - _a4_ _b4_ _c4 d4_
-  - : Are {{cssxref("&lt;number&gt;")}}s describing the translation to apply.
-
-> [!NOTE]
-> Until Firefox 16, Gecko accepted a {{cssxref("&lt;length&gt;")}} value for _a4_, _b4_ and _c4_.
+  - : 表示需要应用平移的 {{cssxref("&lt;number&gt;")}} 值。
```
Comment (yin1999) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:28]:
```suggestion
  - : 描述要应用的平移的 {{cssxref("&lt;number&gt;")}} 值。
```

File: files/zh-cn/web/css/transform-function/matrix3d/index.md [hunk 3]
```diff
@@ -1,92 +1,167 @@
...
 matrix3d(a1, b1, c1, d1, a2, b2, c2, d2, a3, b3, c3, d3, a4, b4, c4, d4)
 ```
 
 ### 值
 
-- _a1_ _b1_ _c1_ _d1_ _a2_ _b2_ _c2_ _d2_ _a3_ _b3_ _c3_ _d3_
-  - : Are {{cssxref("&lt;number&gt;")}}s describing the linear transformation.
+- _a1_ _b1_ _c1_ _d1_ _a2_ _b2_ _c2_ _d2_
+  _a3_ _b3_ _c3_ _d3_
+  - : 表示线性变换的 {{cssxref("&lt;number&gt;")}} 值。
```
Comment (yin1999) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:26]:
```suggestion
  - : 描述线性变换的 {{cssxref("&lt;number&gt;")}} 值。
```

File: files/zh-cn/web/css/transform-function/matrix3d/index.md [hunk 4]
```diff
@@ -1,92 +1,167 @@
...
 
-[CSS](/zh-CN/docs/Web/CSS) 函数 **`matrix3d()`** 以 4x4 齐次矩阵的形式定义一个 3D 转换。其结果是一个 {{cssxref("&lt;transform-function&gt;")}} 数据类型。
+**`matrix3d()`** [CSS](/zh-CN/docs/Web/CSS) [函数](/zh-CN/docs/Web/CSS/CSS_Functions)将 3D 变换定义为一个 4x4 齐次矩阵。其结果是一个 {{cssxref("&lt;transform-function&gt;")}} 数据类型。
+
+{{EmbedInteractiveExample("pages/css/function-matrix3d.html")}}
 
 ## 语法
 
-`matrix3d()` 函数由 16 个参数指定。这些参数以列为主的顺序进行描述。
+`matrix3d()` 函数由 16 个值指定，这些值以列优先顺序排列。
```
Comment (yin1999) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:16]:
```suggestion
`matrix3d()` 函数由 16 个值指定，这些值以列优先顺序进行描述。
```

File: files/zh-cn/web/css/transform-function/matrix3d/index.md [hunk 5]
```diff
@@ -1,92 +1,167 @@
...
+另一个 `transform3d()` 示例，演示了结合平移和缩放的动画效果。
+
+#### HTML
 
 ```html
 <div class="foo">
-  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos quaerat sit
-  soluta, quisquam exercitationem delectus qui unde in facere necessitatibus aut
-  quia porro dolorem nesciunt enim, at consequuntur aliquam esse?
+  那时候他认为必需与重要的是接触大自然，接触曾经生活过、思想过、接触过的前人（如接触哲学、诗歌），现在认为必需、与重要的是人为的规章制度，与跟同事们交往。
```
Comment (yin1999) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:152]: 这段话摘录自哪里（还有别字——必需？）

Comment (T34-active) [inline files/zh-cn/web/css/transform-function/matrix3d/index.md:152]:
```suggestion
  那时候他认为必需与重要的是接触大自然，接触曾经生活过、思想过、接触过的前人（如接触哲学、诗歌），现在认为必需、与重要的是人为的规章制度，与跟同事们交往。——列夫·托尔斯泰《复活》
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**